### ISSUE TITLE
Error during install

### ISSUE THREAD

**Issue by jasazick** [codaamok/PoSH#3]
- Ran script with parameters: .\Posh\AutomatedLab\CM-1902.ps1 -Domain "acc.lab" -SiteCode "ABC"
- SQL ISO added automatically

received the following error:
10:53:26|01:39:12|00:00:00.000|       - Installing Reporting Service Point
10:53:26|01:39:12|00:00:00.343|         - ..
10:53:29|01:39:15|00:00:03.359|         - Failed to install Reporting Service Point (The SQL Server Reporting Server instance name 'SSRS' was not found.)
System.Management.Automation.ActionPreferenceStopException: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The SQL Server Reporting Server instance name 'SSRS' was not found.

**Comment by codaamok** [comment]
Hey

So, just to double check, you're using the latest copy of [CM-1902.ps1](https://github.com/codaamok/PoSH/blob/master/AutomatedLab/CM-1902.ps1) right?

Also, are there any other errors in the console output?

**Comment by codaamok** [comment]
In fact, not even just CM-1902.ps1, all the files in CustomRoles\\* too. 

https://github.com/codaamok/PoSH/tree/master/AutomatedLab/CustomRoles/CM-1902

**Comment by jasazick** [comment]
I deleted and repulled your code from scratch. 
Scrolling through the console output I see two more bits of red spaghetti. 

Get-Content : Cannot find drive. A drive with the name 'D' does not exist.
At C:\Program Files\WindowsPowerShell\Modules\AutomatedLab\5.17.0\AutomatedLabSQL.psm1:174 char:49
+ ...      $configurationFileContent = Get-Content D:\SqlConfig.ini | Conve ...
+                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:String) [Get-Content], DriveNotFoundException
    + FullyQualifiedErrorId : DriveNotFound,Microsoft.PowerShell.Commands.GetContentCommand

10:06:04|00:51:50|00:01:09.896|   - Waiting for installation of SQL server to complete on machines 'CM01'.........................
10:12:12|00:57:58|00:07:18.713|   - .......
10:12:53|00:58:39|00:07:59.332|   - All SQL Servers 'CM01' have now been installed and restarted. Waiting for these to be ready...
10:12:57|00:58:43|00:08:03.657|   - Installing SSRS on''
10:12:57|00:58:43|00:08:03.673|   - Done

**and further up:**

09:43:42|00:29:28|00:00:00.109|   - Downloading .net Framework 4.8 from 'https://download.visualstudio.microsoft.com/download/pr/7afca223-55d2-470a-8edc-6a1739ae3252/abd170b4b0ec15ad0222a809b761a036/ndp48-x86-x64-allos-enu.exe'
09:43:42|00:29:28|00:00:00.124|   - Starting machines 'CM01'....................................................................................................New-LabPSSession : [CM01] Connecting to remote server CM01 failed with the following error message : The WinRM client cannot process the re
quest. The authentication mechanism requested by the client is not supported by the server or unencrypted traffic is disabled
in the service configuration. Verify the unencrypted traffic setting in the service configuration or specify one of the authentication mechanisms supported by the server.  To use Kerberos, specify the computer name as the remote destination. Also verify that the
client computer and the destination computer are joined to a domain. To use Basic, specify the computer name as the remote destination, specify Basic authentication and provide user name and password. Possible authentication mechanisms reported by server:
Negotiate Kerberos For more information, see the about_Remote_Troubleshooting Help topic.
At C:\Program Files\WindowsPowerShell\Modules\AutomatedLab\5.17.0\AutomatedLab.psm1:3405 char:22
+ ... session = @(New-LabPSSession -ComputerName $machines -DoNotUseCredSsp ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (System.Manageme....RemoteRunspace:RemoteRunspace) [Write-Error], PSRemotingTransportException
    + FullyQualifiedErrorId : AccessDenied,PSSessionOpenFailed
Write-LogFunctionExitWithError : Invoke-LabCommand...leaving: Could not create a session to machine 'CM01'
At C:\Program Files\WindowsPowerShell\Modules\AutomatedLab\5.17.0\AutomatedLab.psm1:3408 char:13
+             Write-LogFunctionExitWithError "Could not create a sessio ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-LogFunctionExitWithError

..
10:01:10|00:46:56|00:17:28.103|   - Waiting for pre-requisite .Net 3.5 Framework to finish installation on machines 'CM01'.....Installing software package 'C:\LabSources\SoftwarePackages\vcredist_x86_2015.exe' on machines 'CM01'
10:02:23|00:48:09|00:00:00.938|     - Copying files and initiating setup on 'CM01' and waiting for completion....
10:02:53|00:48:39|00:00:31.537|     - Installation done

**Comment by codaamok** [comment]
So I fixed that hard coded path of `D:\SqlConfig.ini` in the AutomatedLab code a little while ago: https://github.com/AutomatedLab/AutomatedLab/issues/793

Although I can't remember, if the fix is in version 5.17.0 or not. What is your AutomatedLab module version?

**Comment by jasazick** [comment]
5.1.7.0 installed from the MSI installer

**Comment by codaamok** [comment]
Do you mean to say 5.1.7.0? What does `Get-Module AutomatedLab -ListAvailable` tell you?

**Comment by jasazick** [comment]
Bah that was a typo, sorry - , I've got 5.17.0

ModuleType Version    Name                                ExportedCommands                                                                                                        
---------- -------    ----                                ----------------                                                                                                        
Script     5.17.0     AutomatedLab                        {New-LabSourcesFolder, Add-LabAzureSubscription, Add-LabCertificate, Add-LabVMUserRight...}

**Comment by codaamok** [comment]
I just checked out the code in 5.17.0 and it looks like the issue is still in there. No doubt it will be fixed in the next release of AutomatedLab. I'm pretty sure the release is imminent too.

I should have waited before making that blog post. I made the rookie mistake of fixing the code in my installed module and forgetting about it.

I'll keep this issue open and ping you when an AuotmatedLab update is released.
